tools/residency_handoff/validation.py

```
@@ -193,7 +248,94 @@ def changed_paths(self, parent: str, commit: str) -> list[str]:
+            tempfile.TemporaryDirectory(
+                prefix="residency-handoff-archive-repository-"
+            ) as archive_directory,
+            tempfile.TemporaryDirectory(
+                prefix="residency-handoff-empty-git-template-"
+            ) as template_directory,
+        ):
+            archive_repository = Path(archive_directory)
+            environment["GIT_CONFIG_GLOBAL"] = str(
+                Path(template_directory) / "global-config"
+            )
+            initialized_archive = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)